packages/disploy/src/structs/GuildVoiceChannel.ts

```diff
@@ -17,8 +17,7 @@ export class GuildVoiceChannel extends BaseChannel {
 	 * The type of the channel.
 	 */
 	public type: ChannelType.GuildVoice = ChannelType.GuildVoice;
-
-	public constructor(app: App, raw: APIGuildChannel<ChannelType.GuildVoice>) {
+	public constructor(app: App, public override raw: APIGuildChannel<ChannelType.GuildVoice>) {
```

Comment:
While this would probably be a good idea to let the user access the raw objects; it is out of scope for this PR.